csp-guru/csp-guru.csv

```diff
@@ -1,110 +1,112 @@
-Zhangjiakou,,construction,China,,,,,,Fresnel,50.0,nan,,,,,,,,Beijing TeraSolar,,,FIT,1.15,0.1830,0.1649,,,,,,,,,,http://social.csptoday.com/tracker/projects/map?status%5B%5D=Construction; http://en.cspplaza.com/progress-statistics-of-large-scale-commercial-csp-projects-in-china.html;http://en.cspplaza.com/china-ndrc-formally-releases-official-notice-on-benchmark-feed-in-tariff-policy-of-csp-industry.html
-Abijheet,,construction,India,Phalodi,26.8278,70.9197,1985,2013,Trough,50.0,0.0,,Wet,,,,,,Corporate Ispat Alloys,Siemens,Ener-t Internationa,PPA,12.24 INR/kWh,0.1909,0.1720,,,,,,Corporate Ispat Alloys,,,254,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=254
-Diwakar,,construction,India,Askandra,27.3571,71.6918,1938,2013,Trough,100.0,4.0,,Wet,,,,,,Lanco Solar,,Sener,PPA,10.05 INR/kWh,0.1568,0.1413,,,,,,Lanco Solar,,,258,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=258; SUNDERASAN 2015
-KVK,,construction,India,Askandra,27.3819,71.7731,1940,,Trough,100.0,4.0,,Wet,,,,,,KVK,,Sener,PPA,11.1 INR/kWh,0.1731,0.1560,,,,,,KVK,,,260,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=260; SUNDERASAN 2015
-Agua Prieta II,,construction,Mexico,Agua Prieta,31.3255,-109.5471,2799,2011,Trough,12.0,0.0,85000,,34.0000,0.3234,,,,Abengoa,,Abengoa,,,,,,,,,Global environmental facility,Federal Electricity Commission,,,135,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=135
-Stillwater GeoSolar,,construction,USA,Fallon,39.5429,-118.5619,2558,2014,Hybrid; Trough,17.0,0.0,,,3.0000,0.0201,,,,Enel Green Power,,SkyFuel ,PPA,,,,,,,,,Enel Green Power,20.0000,,4279,http://www.nrel.gov/csp/solarpaces/project_detail.cfm/projectID=4279
+Power_station,Year_operational,Status,Country,City,Lat,Lon,DNI_[kWh/m2/yr],Construction_start,Technology,Capacity_[MW],Storage_capacity_[h],Field_size_[m2],Cooling,Expected_generation_[GWh/yr],Load_factor,Total_cost_[million_currency;non-deflated],Total_cost_[million_USD;non-deflated],Total_cost_[million_USD2017;deflated],Specific_investment_cost_[USD2017/kW;deflated],Total_cost_[million_EURO;non-deflated],Total_cost_[million_EURO2017;deflated],Specific_investment_cost_[EURO2017/kW;deflated],Developer,EPC,HCE_manufacturer,SCA_manufacturer,Financed_through,Remuneration_[currency/kWh],Remuneration_[USD/kWh;non_deflated],Remuneration_[USD2017/kWh;deflated],Remuneration_[EURO/kWh;non_deflated],Remuneration_[EURO2017/kWh;deflated],LCOE_5_25_[USD2017],LCOE_10_25_[USD2017],LCOE_5_25_[EURO2017],LCOE_10_25_[EURO2017],Concessional_funding;other_support,Owner,PPA_or_support_duration_[yr],Sources
```

Comment:
The whole file has a weird encoding which destroys special symbols like "€". Excel exports UTF-8, which should be fine. Hence, we should export from Excel again.

Reply:
we solved that together yesterday, right? I have done no changes to the csv.

Reply:
fixed